src/specs.md

```diff
@@ -13,6 +13,41 @@ GNU Affero General Public License for more details.
 You should have received a copy of the GNU Affero General Public License
 along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
+```act
+behaviour wards of MkrAuthority
+interface wards(address usr)
+
+for all
+
+  May : uint256
+
+storage
+
+  wards[usr] |-> May
+
+iff
+
+  VCallValue == 0
+
+returns May
+```
+
+```act
+behaviour root of MkrAuthority
+interface root()
+
+types
```

Comment:
`for all` was used above, here it's `types`...let's be consistent within a given file/repo, at least. I don't have a strong preference, I just care about consistency.

Reply:
should be fixed